tests/test_rse.py

```diff
@@ -572,12 +572,10 @@ def test_add_protocol(self, vo, rucio_client):
             rucio_client.add_protocol(protocol_rse, p)
         resp = mgr.get_rse_info(rse=protocol_rse, vo=vo)
         for p in resp['protocols']:
-            if ((p['port'] == 19) and (p['domains']['lan']['read'] != 1)) or \
-                    ((p['port'] == 20) and (p['domains']['lan']['read'] != 2)) or \
-                    ((p['port'] == 18) and (p['domains']['lan']['read'] != 1)) or \
-                    ((p['port'] == 17) and (p['domains']['lan']['read'] != 4)):
-                print(resp)
-                assert False
+            assert not (((p['port'] == 19) and (p['domains']['lan']['read'] != 1)) or
```

Comment:
While logically equivalent, this change of assert cannot be considered purely cosmetic, and should not be in this commit - should be in 26c459c3915f444b658c84ed8a818912e226307a